File: Java/level 3/Caesar password.java
class Caesar {
	String caesar(String s, int n) {
    String answer = "";
    int temp = 0;
    
    n = n % 26;
    for(int i=0; i<s.length(); i++) {
      char tem = s.charAt(i);
      	if(tem == ' ') {
					answer+=' ';
					continue;
				}
        temp = tem+n;
        if((tem >= 'a') && (tem <='z')) {
            if(temp > 'z') 
              temp -= 26;
        }
        else if((tem >= 'A') && (tem <='Z')) {
            if(temp > 'Z')   
              temp -= 26;
        }
        answer+=(char)temp;  
    }  
    return answer;
	}

	public static void main(String[] args) {
		Caesar c = new Caesar();
		System.out.println("s는 'a B z', n은 4인 경우: " + c.caesar("a B z", 4));
	}
}

File: Javascript/level 3/Caesar password.js
function caesar(s, n) {
	var answer = ""
  n = n % 26 
  for ( let i = 0; i < s.length; i++ ) { 
    var word = s.charCodeAt(i) 
    if ( word == 32 ){
      answer +=" "
    	continue
    }
    var tem = word+n
    if ( word >= 97 && word <= 122 ){ 
      if ( tem > 122 )
        tem-=26
    }
    else if ( word >= 65 && word <= 90 ){
      if ( tem > 90 )  {
        tem-=26
      }
    }
    answer += String.fromCharCode(tem)
  } 
  return answer
}

// 실행을 위한 테스트코드입니다.
console.log('s는 "a B z", n은 4인 경우: ' + caesar("wVLREnMjHgsMCQKGmyipVH tanVgQhwIeKHHey cDXlP OyRVM", 42));